README.md

```diff
@@ -53,6 +53,32 @@ Options:
     output usage information
 ```
 
+### Usage with pre-commit Hooks
+
+Using [lint-staged](https://github.com/okonet/lint-staged) and [husky](https://github.com/typicode/husky), you can lint
+your staged GraphQL schema file before you commmit.  First, install these packages:
+
+```bash
+yarn add --dev lint-staged husky
+```
+
+Then add a `precommit` script and a `lint-staged` key to your `package.json` like so:
+
+```json
+{
+  "scripts": {
+    "precommit": "graphql-schema-linter"
```

Comment:
Do they not need to specify the path here too? i.e. `**/*.graphql`?

Reply:
Whoa, late night coding!  This should be `lint-staged`. :)